features/frameworks/COMPONENT_FPGA_CI_TEST_SHIELD/source/MbedTester.cpp

```diff
@@ -102,8 +103,8 @@ static uint8_t spi_transfer(mbed::DigitalInOut *clk, mbed::DigitalInOut *mosi, m
 static void mbed_tester_command(mbed::DigitalInOut *clk, mbed::DigitalInOut *mosi, mbed::DigitalInOut *miso, uint8_t miso_index, uint32_t addr, bool write_n_read, uint8_t *data, uint8_t size)
 {
     // 8 - Start Key
-    for (uint32_t i = 0; i < sizeof(KEY); i++) {
-        spi_transfer(clk, mosi, miso, KEY[i]);
+    for (uint32_t i = 0; i < sizeof(key); i++) {
```

Comment:
`sizeof` variable name is generally considered bad practice. 
As we are about changing macros, could be also define a macro/const for the size of that array 
`const size_t key_len = 8;` and use it in the array def and the for loops ?

Reply:
Done